blueocean-dashboard/src/main/js/components/BranchDetailsRow.jsx

```diff
@@ -84,7 +77,7 @@ export class BranchDetailsRowRenderer extends Component {
                 <TableCell linkTo={runDetailsUrl}><CommitId commitId={commitId} /></TableCell>
                 <TableCell>{ runMessage }</TableCell>
                 <TableCell linkTo={runDetailsUrl}>{ completed }</TableCell>
-                { actionsCell }
+                <TableCell className="TableCell--actions">{ actions }</TableCell>
```

Comment:
why does this css class have a double dash in it? I know you didnt add it but seems weird.

Reply:
I am not a fan of double dashes either tbh, because it is part of the JTable components I think it was @sophistifunk decision to name it like this, I don't really mind it but I agree it looks a little strange

Reply:
Just trying to follow [a standard](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md).

Although I think I'm leaning more towards [this](http://ecss.io/).